content/blog/2020-12-10-masks-public.Rmd

```diff
@@ -0,0 +1,398 @@
+> 2. Most of the people were wearing masks
+> 3. Some of the people were wearing masks
+> 4. A few of the people were wearing masks
+> 5. None of the people were wearing masks
+> 6. I have not been out in public places in the past 7 days
+
+Because this question asks about *other* people, it bypasses social desirability
+bias. And even if the survey sample is not representative, as long as the
+respondents visit representative public places (like grocery stores or public
+transit) and see *other* people there, we may get a better estimate of public
+mask use.
+
+Let's see how the data looks. We'll plot the percentage of people answering
+"all" or "most" to each mask question in each state, with a diagonal line
+indicating perfect agreement between the numbers:
+
+```{r mask_questions_compared, message=FALSE}
+masked <- covidcast_signal("fb-survey", "smoothed_wwearing_mask",
+                           start_day = "2020-12-01", end_day = "2020-12-01",
+                           geo_type = "state")
+other_mask <- covidcast_signal("fb-survey", "smoothed_wothers_masked",
+                               start_day = "2020-12-01", end_day = "2020-12-01",
+                               geo_type = "state")
+
+joined <- masked %>%
+    inner_join(other_mask, by = "geo_value", suffix = c(".self", ".other"))
+
+ggplot(joined, aes(x = value.self, y = value.other,
+                   label = toupper(geo_value))) +
+  geom_text() +
+  geom_abline(slope = 1, intercept = 0) +
+  labs(x = "% who report wearing masks most/all the time",
+       y = "% who report most/all others wear masks",
+       title = "Mask use reported in symptom survey",
+       subtitle = "December 1st, 2020",
+       caption = "Data from Delphi COVIDcast, delphi.cmu.edu") +
+  theme_bw()
+```
+
+There are a few things worthy of note. There is clearly a strong relationship
+here---but notice that in South Dakota and Wyoming, where over 80% report they
+wear masks most or all of the time, fewer than half of respondents agree that
+most or all *other* people wear masks in public places. Only in a few states do
+over 90% of respondents agree that most other people wear masks:
+
+```{r}
+joined %>%
+  filter(value.other >= 90) %>%
+  arrange(desc(value.other)) %>%
+  select(geo_value, value.other) %>%
+  mutate(geo_value = abbr_to_name(geo_value, ignore.case = TRUE)) %>%
+  knitr::kable(col.names = c("State", "% agree that most/all others are masked"),
+               digits = 1)
+```
+
+But we should also be careful in interpreting this. In South Dakota and Wyoming,
+fewer than half of respondents agree that most or all other people wear masks.
+That's not the same thing as saying that fewer than half of people wear masks.
+For example, if people interpret "most people wear masks" to mean "I only see
+one or two people not wearing masks when I'm in public", "most" could correspond
+to 80 or 90% mask use, rather than just 50%; it's not obvious how to translate
```

Comment:
```suggestion
to 80 or 90% mask use, rather than just 50%. It's not obvious how to translate
```
Makes it easier to read, but again disregard if it changes any meaning.